lib/build.js

```diff
@@ -22,7 +22,35 @@ log.level = config.get('log_level');
  */
 function _readConfig(configFile, callback) {
   log.info('closure-util', 'Reading build config');
-  buildconfig.readConfig(configFile, callback);
+  configfile.readConfig(configFile, callback);
+}
+
+
+/**
+ * Assert that a provided config object is valid.
+ * @param {Object} config Build configuration object.
+ * @param {function(Error, Object)} callback Called with the configuration
+ *     object or any error.
+ */
+function _assertValidConfig(config, callback) {
```

Comment:
I'm sure you know, but the `_privateStyle` names are not really necessary.  Only properties explicitly assigned to `exports` or `module.exports` are available outside this module.